### DISCUSSION TITLE
Listings API vs. Feeds API for Product Management?

### DISCUSSION THREAD

**Discussion by ShaneOH** [amzn/selling-partner-api-models#2818] [General] [answered]
Hey all, found a related issue here https://github.com/amzn/selling-partner-api-docs/discussions/1756 but it didn't get much attention. 

Given that Listings API seems to have bulk processing support, is there any reason to use the Feeds API over the Listings API? Is there anything the Listings API lacks? Is there any reason to integrate both? 

We already have our product management system built out via the Feeds Management, and we're wondering if it's worth the effort to rebuild via the Listings API. We'd get synchronous support then, and easier maintenance, but we're curious if there are any potential pitfalls, especially at scale. 

Would greatly appreciate any input!

**Comment by wanglei0214** [comment, marked as answer]
Hi ShaneOH,

Thanks for asking.
We provide both Listing API and Feeds API for product management. There is no specific reasons, the Feeds API is the long exist API suite which could submit not only listing information, but also submit other data i.e. order shipping information etc; While the Listing API is the latest launched API suite for Listing management purpose specifically.

So far, Feeds API for product listing management is commonly used for the world wide SP-API developers,
but we would like to recommend you to adopt the latest launched Listing API for future Product Management.

Best,
Lei
